util/cf_target.go

```diff
@@ -0,0 +1,17 @@
+package util
+
+import plugin_models "github.com/cloudfoundry/cli/plugin/models"
+
+type CloudFoundryTarget struct {
+	Org      plugin_models.Organization
+	Space    plugin_models.Space
+	Username string
```

Comment:
is there some kind of predefined plugin model for the user entity?

Reply:
Sadly, no. It just returns a `string`